pyvista/plotting/camera.py

```diff
@@ -649,19 +648,19 @@ def tight(self, padding=0.0, adjust_render_window=True, view='xy', negative=Fals
 
         Parameters
         ----------
-        padding : float, optional
+        padding : float, default: 0.0
             Additional padding around the actor(s). This is effectively a zoom,
             where a value of 0.01 results in a zoom out of 1%.
 
-        adjust_render_window : bool, optional
+        adjust_render_window : bool, default: True
             Adjust the size of the render window as to match the dimensions of
             the visible actors.
 
-        view : {'xy', 'yx', 'xz', 'zx', 'yz', 'zy'}
-            Plane to which the view is oriented. Default 'xy'.
+        view : {'xy', 'yx', 'xz', 'zx', 'yz', 'zy'}, default: 'xy'
+            Plane to which the view is oriented.
```

Comment:
numpydoc only suggests putting the default first in the set.  However I don't like this, your suggestion seems okay to me, but interested in others take.

Reply:
Yeah I think this makes sense